src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -77,7 +94,7 @@ public void Dispose()
             _parametersCache.Dispose();
         }
 
-        private async Task<bool> IsEnabledAsync<TContext>(string feature, TContext appContext, bool useAppContext)
+        private async Task<bool> IsEnabledAsync<TContext>(string feature, TContext appContext, bool useAppContext, bool ignoreVariant)
```

Comment:
I was using it as a flag where it's set to true only in `GetVariantAsync`. It might look confusing since it looks like the true in  `await IsEnabledAsync(feature, context, true)` is setting `ignoreVariant` here, not `useContext`. That did make me realize that I need to set this helper to `private` though.